packages/ui-components/src/README.md

```diff
@@ -2,3 +2,4 @@
 
 - [Badge](Badge/README.md)
```

Comment:
This README is already missing components (avatar and input). I don't think maintaining this list is sustainable either.